metrics/density/docker_memory_usage.sh

```diff
@@ -87,6 +98,34 @@ get_runc_pss_memory(){
         echo "$avg"
 }
 
+get_runc_individual_memory() {
+	get_sequence
+	individual_numbers=$(cat $SEQ_TMP_FILE)
+	runc_process_result=$(cat $MEM_TMP_FILE | tr "\n" " " | sed -e 's/\s$//g' | sed 's/ /, /g')
+
+	# Verify runc process result
+	if [ -z "$runc_process_result" ];then
+		die "Runc process not found"
+	fi
+
+	read -r -a runc_values <<< "${runc_process_result}"
+	read -r -a numbers <<< "${individual_numbers}"
+
+	metrics_json_start_array
+
+	local json="$(cat << EOF
+	{
+		"runc individual results": {
```

Comment:
I'd prefer not having spaces in json keys.

Reply:
@jcvenegas this is the way that we have been doing the `json` for metrics

Reply:
@jcvenegas all metrics have the same format

Reply:
I'm not sure if that is strictly true ;-), but I believe we have got some and they are creeping in more. I don't have a distinct problem with spaces in the identifiers - as they are quoted we can parse them out OK I believe. I do suspect we can mostly clearly define the data by using appropriately named sub sections in the JSON structure - so, let's keep questioning ourselves and trying to structure the data so it is clear and makes sense etc.